### ISSUE TITLE
Fix item duplication if possible

### ISSUE THREAD

**Issue by DustyBagel** [DumbDave961/technic_elepower_compat#7]
Because of having elepwer and technic together, some infinite item duplication methods are possible.  If possible, we should find a way to get rid of that so that the modpack is able to be used on public servers at some point.

**Comment by DumbDave961** [comment]
I did deal with the plate duplication bug. Are there any others we should worry about?

**Comment by DustyBagel** [comment]
Well there is the sandstone duplication thing.

**Comment by DumbDave961** [comment]
We might be able to fix that if we register the recipe with reduced grinding time, but I haven't tested it yet.

**Comment by DumbDave961** [comment]
I tested your example for clearing the zinc plate recipe for the Elepower compressor, but it didn't work. I can't clear the grinding recipe either. None of the other recipes I tested were cleared. Are you sure you tested both the recipe clearing functions you made?

**Comment by DustyBagel** [comment]
Yes, I tested them. I tried clearing them again in a new download of the repo and it didn't work for some reason, but I fixed it in pull request #17 as well as some other improvements. Remember when using the elepower_clear_recipe function the item string that you are trying to clear needs to be exactly the same as what it is registered as. For example, if you wanted to clear the recipe for silver_sand you would need to do it like the below example because in the elepower table that the recipe is in has the item output number in it.

`elepower_clear_craft("grind", "defualt:silver_sand 4")`
